src/navigation/StackNavigators/ObservationsStackNavigator.js

```diff
@@ -0,0 +1,74 @@
+// @flow
+
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { createNativeStackNavigator } from "@react-navigation/native-stack";
+import MyObservationsContainer from "components/MyObservations/MyObservationsContainer";
+import DataQualityAssessment from "components/ObsDetails/DataQualityAssessment";
+import ObsDetailsContainer from "components/ObsDetails/ObsDetailsContainer";
+import TaxonDetails from "components/TaxonDetails/TaxonDetails";
+import UserProfile from "components/UserProfile/UserProfile";
+import { t } from "i18next";
+import {
+  blankHeaderTitle,
+  hideHeader,
+  removeBottomBorder,
+  showHeaderLeft,
+  showLongHeader
+} from "navigation/navigationOptions";
+import type { Node } from "react";
+import React from "react";
+
+const Stack = createNativeStackNavigator( );
+
+const ObservationsStackNavigator = ( ): Node => (
```

Comment:
Is the intent to reuse this navigator in Explore? If not, can we call it the `MyObservationsStackNavigator`? Consider what should happen when the user taps the Explore button in the header of MyObservations. I don't think that should actually open Explore within the same stack, i.e. pushing a new window in from the right and allow the user to pop back out to My Obs. Instead, I think it should be like popping out to the root and entering a new stack, albeit with some params.

Reply:
Ok, @abhasinat and I [discussed this](https://inaturalist.slack.com/archives/C02G5AATVGA/p1692740155221459) and he made a good case for a combined stack because there are so many ways to get to explore (from MyObs, from user profile, from taxon detail...), so `ObservationsStackNavigator` it is!